src/components/Requirements.vue

```diff
@@ -1,26 +1,276 @@
+            {
+              name: "Quantitative Literacy",
+              count: "Credits",
+              progress: 2,
+              total: 2
+            },
+            {
+              name: "Chemistry/Physics",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ]
+        },
+        {
+          name: "Major Requirements",
+          type: "Major",
+          count: "Courses",
+          progress: 4,
+          total: 8,
+          color: '#4D53DC',
+          displayDetails: false,
+          todo: [
+            {
+              name: "CALS Credits",
+              count: "Credits",
+              progress: 12,
+              total: 55
+            },
+            {
+              name: "PE Credits",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ],
+          displayCompleted: true,
+          completed: [
+            {
+              name: "Quantitative Literacy",
+              count: "Credits",
+              progress: 2,
+              total: 2
+            },
+            {
+              name: "Chemistry/Physics",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ]
+        }
+      ]
+    }
+  },
+
+  methods: {
+    turnDetails: function(index, bool) {
+      this.reqs[index].displayDetails = bool;
+    },
+    turnCompleted: function(index, bool) {
+      this.reqs[index].displayCompleted = bool;
+    }
+  }
+}
 </script>
 
 <style scoped lang="scss">
+
+.btn {
+  padding: 0;
+}
+
 .requirements {
-  width: 25rem;
-  background-color: lightblue;
-  border: 2px black solid;
-
-  &-ayesha {
-    width: 25rem;
-    font-size: 50px;
-    color: black;
-    text-align: center;
+  width: 30rem;
+  padding: 1.625rem 1.5rem 1.625rem 1.5rem;
+  background-color: white;
+}
+
+h1.title {
+  margin: 0 0 0 0;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 24px;
+}
+
+.top {
+  margin: 1.5rem 0 1rem 0;
+}
+
+.name {
+  margin-top: auto;
+  margin-bottom: auto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 16px;
+}
+
+.settings {
+  height: 14px;
+  width: 14px;
+  fill: #999999;
+}
+
+.progress-text {
+  margin: 0.3125rem 0 0 0;
+  font-size: 12px;
+  line-height: 12px;
+}
+
+button.view {
+  margin: 0.7rem 0 2rem 0;
+  min-height: 40px;
+  width: 100%;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 14px;
+  text-align: center;
+  color: white;
+  text-transform: uppercase;
+}
+
+.detail-bar {
+  margin: 1.625rem 0 0.8125rem 0;
+  width: 100%;
+}
+
+.detail-text {
+  margin-top: auto;
+  margin-bottom: auto;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 14px;
+}
+
+.cancel {
+  height: 10px;
+  width: 10px;
+}
+
+.toggle {
+  margin-top: auto;
+  margin-bottom: auto;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 12px;
+}
+
+.req {
+  margin: 0.5rem 0 0 0;
+  padding: 0;
```

Comment:
Why?

Reply:
I was trying to override a bootstrap style. Pretty sure it didn't work and I forgot to remove it